Make.inc.in

```diff
@@ -184,3 +184,15 @@ PANDOC_ENGINE := @PANDOC_ENGINE@
 DOXYGEN := @DOXYGEN@
 GRAPHVIZ_DOT := @GRAPHVIZ_DOT@
 ANTORA := @ANTORA@
+
+#
+#  You can watch what it's doing by:
+#
+#	$ VERBOSE=1 make ... args ...
```

Comment:
this block is already in `scripts/boiler.mk`.  It shouldn't be needed here.

Reply:
Ah! you're right, I will remove that snip.

Reply:
@alandekok done